### PR TITLE
Artwork unit test

### PR DESCRIPTION
**Pull request by JeffersonMat**
Link to associated zer0 card:  
[Card](https://www.notion.so/rational-nomads/FEATURE-Tests-for-Artwork-component-b75f374f093444778ee877bfe3e1bcab)
-----

## 1. Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] zer0 card has been moved to the Code Review column
- [x] zer0 card has a link to this PR
- [x] A reviewer has been tagged in the zer0 card

## 2. PR type

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## 3. What is the current behaviour?

The Artwork component didn't have a unit test

## 4. What is the new behaviour?

Created unit test for Artwork

### CODE REVIEW

**Comment by padala** [comment]
@JeffersonMat  Is this PR ready for review?

File: package.json [hunk 1]
```diff
@@ -83,7 +83,7 @@
 	"scripts": {
 		"start": "react-scripts start",
 		"build": "npm run generate-meta-tag && react-scripts build",
-		"test": "react-scripts test",
+		"test:watch": "react-scripts test",
```
Comment (ericlowturing) [inline package.json:86]: Do we need to change this script name?

**Review by ericlowturing — APPROVED** [review]
LGTM 💯 . Approving with pending comment about the script name. The circleci test is failed. Need to be get success with circleci.

File: package.json (hunk 1 above)
Comment (JeffersonMat) [reply package.json:86]: @ericlowturing not really, I changed it so I could watch the test a bit cleaner. I'll remove it now.  
Thank you!

**Pull request merged**